dvc/exceptions.py

```diff
@@ -247,7 +247,7 @@ def __init__(self, out_1, out_2):
 class CheckoutErrorSuggestGit(DvcException):
     def __init__(self, target, cause):
         super(CheckoutErrorSuggestGit, self).__init__(
-            "Did you mean 'git checkout {}'?".format(target), cause=cause
+            "Did you mean `git checkout {}`?".format(target), cause=cause
```

Comment:
This is the only one I'm not sure about.

Reply:
Yeah, this one is a good example where both approaches collide :D In this specific case using ticks to wrap the whole command plus single quotes to write {} specifically would work too, but the way you've formatted is totally correct. Thank you so much! 🙂